examples/helmcharts/nginx/values.yaml

```diff
@@ -195,9 +195,9 @@ ingress:
   ##
   ## It is also possible to create and manage the certificates outside of this helm chart
   ## Please see README.md for more information
-  # - name: example.local-tls
-  #   key:
-  #   certificate:
+   - name: example.local-tls
```

Comment:
Rolling back changes

Reply:
I am only uncommenting values for maximum code coverage. The template won't be enabled since there is an if condition at the top of the template. There is no side effects.

Reply:
Per offline discussion, we decided to keep this uncommented to ease demoing the for-loop conversion process.  In reality, we will likely enable pulling from a URL instead in the future for upstream charts. We acknowledge that this changes the default nginx chart, but not in a way that is destructive.  This purely makes it a bit easier to showcase how the for-loop conversion works.